### PR TITLE
checking Feature branch for stickler activation.

### PR DESCRIPTION
**Pull request by jamestucker1988**
xyz

### CODE REVIEW

**Comment by jamestucker1988** [comment]
-The given project is a curriculum-based project. All the objectives outlined in the instructions are achieved- namely, using grid,flex-box,float to clone the newyork times webpage. 
-The page consists of a navbar,body with article tags and a footer section .

File: index.html [hunk 1]
```diff
@@ -1,26 +1,335 @@
...
+                <li>Terms of Service</li>
+                <li>Site Map</li>
+                <li>Help</li>
+                <li>Subscriptions</li>
+                <li>SaleSite</li>
+            </ul>
+
+        </div>
+        <div class="underline"></div>
+    </div>
```
Comment (lucasmazo32) [inline index.html:334]: There is an element that is missing and is the bottom section of the page (that sticks to the bottom and says expand).

I am attaching a screenshot from the original page.

image

File: index.html [hunk 2]
```diff
@@ -1,26 +1,335 @@
...
 </body></html>
```
Comment (lucasmazo32) [inline index.html:335]: There are some warnings in the HTML validator. Kindly fix them.

> 
image

File: index.html [hunk 3]
```diff
@@ -1,26 +1,335 @@
...
+            <li class="nav-left"><span class="fa fa-search
+                "></span></li>
+            <li class="nav-left">space&amp;cosmos</li>
+        </ul>
+        <img src="image/newyork.svg" alt="unable to load">
+        <ul class="nav-ul">
+            <li><button>Subscribe Now</button></li>
             <li><button>log in</button></li>
         </ul>
     </nav>
```
Comment (lucasmazo32) [inline index.html:29]: The navbar doesn't look like the one on the original webpage.

I will attach a screenshot. 

> 
image

File: index.html [hunk 4]
```diff
@@ -1,26 +1,335 @@
...
+    <div class="main-content">
+        <strong>By Dennis Overbye</strong>
+        <div class="date">
+            <p>March 17, 2014</p>
+            <ul>
+                <li><i class="fab fa-facebook"></i></li>
+                <li><i class="fab fa-twitter-square"></i></li>
+                <li><i class="fas fa-envelope"></i></li>
+                <li><i class="fas fa-share"></i></li>
+                <li> <i class="medium material-icons">bookmark</i></li>
```
Comment (lucasmazo32) [inline index.html:44]: These icons should be links. You can wrap the icon inside an anchor tag (```<a href="#"><i></i></a>```).

File: index.html [hunk 5]
```diff
@@ -1,26 +1,335 @@
...
+            <p><strong>INFLATION</strong><br>solves this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.</p>
+            <p><strong>FLUCTUATION</strong><br> Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.</p>
+            <p><strong>EXPANSION</strong><br>Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.</p>
+        </div>
```
Comment (lucasmazo32) [inline index.html:77]: This text sometimes is over the images. Please, fix that.

> 
image

File: index.html [hunk 6]
```diff
@@ -1,26 +1,335 @@
...
+            <p>A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun.<br> <span>Order Reprints</span><span> Today’s Paper</span><span> Subscribe</span>
+            </p>
+            <div class="footer1">
+                <button>READ 615 COMMENTS</button>
+
+                <ul>
+                    <li><i class="fab fa-facebook "></i></li>
+                    <li><i class="fab fa-twitter-square "></i></li>
+                    <li><i class="fas fa-envelope "></i></li>
+                    <li><i class="fas fa-share "></i></li>
```
Comment (lucasmazo32) [inline index.html:148]: These icons should be links as well.

File: index.html [hunk 7]
```diff
@@ -1,26 +1,335 @@
...
+                <div>Finally There’s Time to Read (or at Least Organize Your Books)</div>
+                <div>A Father’s Love, Whether He Says Too Much or Not Enough</div>
+                <div>Polls Had Trump Stewing, and Lashing Out at His Own Campaign</div>
+                <div>De Blasio Breaks Up Rabbi’s Funeral and Lashes Out Over Virus Distancing</div>
+                <div>Why Zoom Is Terrible</div>
+                <div>This Florida Student Was Accepted at All 8 Ivy League Schools</div>
+                <div>Opinion: At Least 89 Vaccines Are Being Developed. It May Not Matter.</div>
+                <div>What Phone Calls Have Given Me That Video Chat Can’t</div>
+                <div>Top E.R. Doctor Who Treated Virus Patients Dies by Suicide</div>
+                <div>Opinion: They Didn’t Drink the Bleach, but They’re Still Drinking the Kool-Aid</div>
```
Comment (lucasmazo32) [inline index.html:217]: This text looks different from the original. Kindly fix that.

Original
> 
image

Your page
> 
image

File: style.css [hunk 8]
```diff
@@ -0,0 +1,379 @@
...
+.alan_guth {
+  position: relative;
+  align-self: center;
+}
+
+.main-content {
+  position: relative;
+  padding-top: 8rem;
+
+  /*  padding-left: 5rem; */
```
Comment (lucasmazo32) [inline style.css:151]: Please erase this commented code.

**Comment by jamestucker1988** [comment]
I introduced the changes recommended by TSE,namely
1-remove validation warnings.
2-modified nav element.
3-added anchor tags to the social icon.
4- fixed background image in grid-container.
5-changing texts "Due to dynamic rendering of webpage the contents change everytime  u access or few hours I have when I first accessed it."
6-erased the commented code.

**Comment by jamestucker1988** [comment]
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
1-remove validation warnings. And indentation errors.
2-modified nav element.
3-Added anchor tags to the social icon.
4- fixed background image in the grid-container.
5-erased the commented code.

**HERE is a preview link:<a  href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">here</a>**

**Comment by jamestucker1988** [comment]
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
-I have changed navigation "Humberger" icon to be a button.
 -changed "SPACE & COSMOS" in the navigation bar should to be a link.
 -removed unnecessary white space above your navigation bar.
 -removed  horizontal scroll bar.
- "More in Space and Astronomy" section has links to other articles. 
- "Most Popular" section has links to more popular articles. 
 -The bottom section with the logo and other navigation links have been made links.
 -the clone has a footer that sticks to the bottom part of the screen. 

**HERE is a live preview <a href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">click here</a>**

**Comment by sumancrest0001** [comment]
**Status: changes required
Hello @jamestucker1988** 
You have done a good job to fix the issues raised by the previous TSEs but we still need to make a couple of changes to get :+1:  for this project. 
**[NOTE] All the requirements are mandatory.**

**Comment by sumancrest0001** [comment]
Let's fix the navbar issue. change the space & cosmos font to match the original one, add a background color to buttons, and other sections are overlapping with the navbar when we scroll.
Have a look :point_down: 
Screenshot from 2020-05-13 08-37-28

**Comment by sumancrest0001** [comment]
The main section should be centered and width should be similar to the original one. currently, your main section is narrower.

The More in Space and Astronomy section should have 3 article column but it's a two-column section in your project. Please consider the following image to get clear.
Screenshot from 2020-05-13 08-42-00

**Comment by sumancrest0001** [comment]
Let's fix the alignment of items in the following sections of the footer. They should be aligned at left.
Screenshot from 2020-05-13 08-39-59

But currently :point_down: 
Screenshot from 2020-05-13 08-54-26

**Comment by sumancrest0001** [comment]
we have to fix  the overlapping problem with the sticky footer section also. 
[TIPS] Add background color to white.

**Comment by jamestucker1988** [comment]
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
1-fixed overlapping problem with sticky footer.
2-alignment items in footer section.
3-more in space and astronomy now has three columns.
4-changed fontsize,color in navbar.

**HERE is a preview link:<a  href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">here</a>**

File: index.html [hunk 9]
```diff
@@ -1,26 +1,340 @@
...
     <nav>
+        <div class="nav-ul button1">
+            <button><i class="fas fa-bars"></i></button>
+            <i class="fa fa-search"></i>
 
+            <a href="#">space &amp; cosmos</a>
 
+        </div>
+        <img src="image/newyork.svg" alt="unable to load">
+        <div class="nav-ul">
```
Comment (lucasmazo32) [inline index.html:24]: - [ ] Kindly wrap the logo inside an anchor tag `<a></a>` as the original page.

File: index.html [hunk 10]
```diff
@@ -1,26 +1,340 @@
...
+                <li><a href="#">Work with us</a></li>
+                <li><a href="#">Advertise</a></li>
+                <li><a href="#">T Brand Studio</a></li>
+                <li><a href="#">Your Ad Choices</a></li>
+                <li><a href="#">Privacy</a></li>
+                <li><a href="#">Terms of Service</a></li>
+                <li><a href="#">Site Map</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Subscriptions</a></li>
+                <li><a href="#">SaleSite</a></li>
```
Comment (lucasmazo32) [inline index.html:327]: - [ ] Please check this section. Some of the words are overlaping.

> image

**Comment by jamestucker1988** [comment]
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
1-wrapped the logo inside an anchor tag <a></a> as the original page.
2-removed word overlapping in the indicated section.

**HERE is a preview link:<a  href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">here</a>**

**Comment by rammazzoti2000** [comment]
## Project Approved :rocket: :tada: 
Hello team great job, your project is approved, feel free to merge whenever you are ready.
###
Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :heavy_check_mark: Status: Approved

**Comment by rammazzoti2000** [comment]
Hello @jamestucker1988, you shouldn't close the pull request but to merge the branch, reopen this pull request and than merge, after which submit your form for the completion of the project
For more clarifications you can find me on slack under the name of alexandru

**Comment by jamestucker1988** [comment]
I am reopening the pull request.

**Comment by jamestucker1988** [comment]
I closed and merged it with the feature branch.
**here is a new pull request I have create<a href="https://github.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/pull/4">click here</a>**

**Pull request closed without merging**